src/components/study-groups/StudyGroupMarketplace.tsx

```
@@ -0,0 +1,183 @@
+                {groups.length === 0 ? (
+                    <div className="mt-8 text-center p-16 bg-slate-900/30 border border-slate-800/80 rounded-3xl border-dashed">
+                        <Users className="w-12 h-12 text-slate-700 mx-auto mb-4" />
+                        <h3 className="text-lg font-bold text-white mb-2">No active groups</h3>
+                        <p className="text-slate-400 text-sm">Be the first to host a study session for your course!</p>
+                    </div>
+                ) : (
+                    <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6">
+                        {groups.map(group => {
+                            const isFull = group.members.length >= group.capacity || group.status === "full";
+                            return (
+                                <div
```

ESLint

[error] Replace `································` with `················`

(prettier/prettier)